README.md

```diff
@@ -6,44 +15,21 @@ The contract responsible for sending/receiving messages from the base layer gate
 
 - [Documentation/Diagrams](./docs/design_docs.md)
 - [Deploying Token Contracts](./docs/deploy_docs.md)
-- [Build From Source](#building_from_source)
-- [Contributing](#contributing)
 - [License](#license)
 
 ### Bridge Message Predicates
 
 This project uses the general contract message relaying script/predicate from the [bridge-message-predicates](https://github.com/FuelLabs/bridge-message-predicates) repo.
 
-## Building From Source
-
-### Dependencies
-
-| dep     | version                                                  |
-| ------- | -------------------------------------------------------- |
-| Forc    | [v0.35.0](https://fuellabs.github.io/sway/v0.35.0/introduction/installation.html) |
-
-### Building
-
-Build:
-
-```sh
-forc build
-```
-
-Run tests:
+## Quickstart
 
-```sh
-cargo test
-```
+### Building Sway
 
-## Contributing
+In the root of the repository run `forc build` to build all the Sway programs. Each Sway project uses a `fuel-toolchain.toml` file with pinned versions so `forc` will install the correct versions for you if you do not have them installed.
```

Comment:
I think, breaking the sections in lines is better to read than adding the commands in the middle of the text.